content/en/opentelemetry/schema_semantics/hostname.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Mapping OpenTelemetry Semantic Conventions to Hostnames
+kind: documentation
+further_reading:
+- link: "/opentelemetry/"
+  tag: "Documentation"
+  text: "OpenTelemetry Support in Datadog"
+---
+
+## Overview
+
+OpenTelemetry defines certain semantic conventions related to host names. If an OTLP payload for any signal type has known hostname attributes, Datadog honors these conventions and tries to use its value as a hostname. The default hostname resolution algorithm is built with compatibility with the rest of Datadog products in mind, but you can override it if needed.
+
+This algorithm is used in the [Datadog exporter][3] as well as the [OTLP ingest pipeline in the Datadog Agent][2]. When using the [recommended configuration][4] for the Datadog exporter, the resource detection processor adds the necessary attributes to the payload to ensure accurate hostname resolution.
+
+## Conventions used to determine the hostname
+
+Conventions are checked in the following order, and the first valid hostname is used. If no valid conventions are present, the fallback hostname logic is used. This fallback logic varies by product.
+
+1. Check the general `host` and `datadog.host.name` Datadog-specific conventions.
+1. Cloud provider-specific conventions for AWS, Azure and GCP.
+1. Kubernetes-specific conventions.
+1. Fall back to `host.id` and `host.name` if no specific conventions are found.
+
+The following sections explain each set of specific conventions in detail.
+
+### General hostname semantic conventions
+
+The `host` and `datadog.host.name` conventions are Datadog-specific conventions. They are considered first and can be used to override the hostname detected using the usual OpenTelemetry semantic conventions. Prefer using the `datadog.host.name` convention since it is namespaced and it is less likely to conflict with other vendor-specific behavior.
```

Comment:
```suggestion
The `host` and `datadog.host.name` conventions are Datadog-specific conventions. They are considered first and can be used to override the hostname detected using the usual OpenTelemetry semantic conventions. Prefer using the `datadog.host.name` convention since it is namespaced, and it is less likely to conflict with other vendor-specific behavior.
```